mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -56,6 +62,16 @@ CLASS none/se net/minecraft/entity/Entity
 		ARG 1 commandName
 	METHOD a setUuid (Ljava/util/UUID;)V
 		ARG 0 uuid
+	METHOD a dropItems (Lnone/aeo;I)Lnone/yy;
```

Comment:
Yes but I kept it this way so it's an overload to the other functions that also take the stack amount.